container/root/etc/nginx/nginx.conf

```diff
@@ -0,0 +1,125 @@
+#############################################################
+# Replace nginx configuration directly in this file,
+# removing the need to perform error-prone replacements
+# at build time.
+#
+# For run-time replacements, ie, consuming environment vars,
+# add to the run.d/nginx script
+#############################################################
+
+user nobody;
+worker_processes auto;
+
+pid /tmp/nginx.pid;
+
+events {
+    worker_connections 1024;
```

Comment:
Are there any filehandle limits, etc. we need to worry about w/ Alpine @bryanlatten ?